src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -78,6 +80,14 @@ public function collect(Request $request, Response $response, \Exception $except
                 $this->data['bundles'][$name] = $bundle->getPath();
             }
         }
+
+        $versionRequest = file_get_contents('http://symfony.com/roadmap.json?version='.$this->data['symfony_version']);
```

Comment:
Hmm, that sounds like a plan. When the oem date changes, it's probably to make the maintance period longer and not shorter. This means when oem is reached, we can retrieve the data again to see if the dates changed.